cypress/e2e/login-form.cy.ts

```diff
@@ -0,0 +1,91 @@
+describe('Login form', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:5173/#/login');
+
+    cy.url().should('eq', 'http://localhost:5173/#/login');
+  });
+
+  describe('should success when submit with', () => {
+    it('valid given credentials', () => {
+      cy.login({
+        username: 'admin',
+        password: 'admin',
+      });
+
+      cy.url().should('eq', 'http://localhost:5173/#/professionals');
+
+      cy.get('.css-jp2pjq-RaUserMenu-root > .MuiButtonBase-root')
```

Comment:
We should try to use test-id or id by Cypress recomendation, we decouple from MUI if they decide to changed the classnames in the future.